Bottom: 155e1c70fe42c2e21f6806583bd6829ad00e2f5a
Top:    056b9d84d0e722a063b7b150bbc824de05884693
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-15 01:13:28 +0000

Refresh of crasy-test

---

diff --git a/src/draw.c b/src/draw.c
index a485ed2..0d0ad6d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -520,8 +520,7 @@ DrawEMark (ElementType *e, Coord X, Coord Y, bool invisible)
       mark_size = MIN (mark_size, pad0->Thickness / 2);
     }
 
-  gui->graphics->set_color (Output.fgGC,
-		  invisible ? PCB->InvisibleMarkColor : PCB->ElementColor);
+  gui->graphics->set_color (Output.fgGC, invisible ? PCB->InvisibleMarkColor : PCB->ElementColor);
   gui->graphics->set_line_cap (Output.fgGC, Trace_Cap);
   gui->graphics->set_line_width (Output.fgGC, 0);
   gui->graphics->draw_line (Output.fgGC, X - mark_size, Y, X, Y - mark_size);
